src/faultlog/poweron_time.cpp

```diff
@@ -0,0 +1,118 @@
+#include "xyz/openbmc_project/Logging/Entry/server.hpp"
+
+#include <phosphor-logging/lg2.hpp>
+#include <poweron_time.hpp>
+
+#include <chrono>
+#include <fstream>
+
+namespace openpower::faultlog
+{
+constexpr auto poweronTimeFile =
```

Comment:
Usually I see file location defined in meson options. Just wanted to know if it better to be here or move to meson options

Reply:
if something does not change permanently we can add it in the code too.